financial-templates-lib/logger/Logger.js

```diff
@@ -0,0 +1,39 @@
+// The logger has a number of different levels based on the severity of the incident:
+//-> Debugs: self explanatory. Normal status-based logging. These can trigger
+//   every iteration. Unlimited volume.
+//-> Info: when something happens that is notable, but not necessarily actionable.
+//   These should not trigger every iteration. Any on-chain event that executed correctly.
+//   these trigger a slack message that everyone has access to.
+//-> Error: anything that requires human intervention. If the bot is low on funds or a
```

Comment:
Question for you: Does logger.error also print out errors? I've noticed when running the `liquidator/test/liquidator.js` tests, the last test correctly catches an error but also prints out the error which may not be as intended?

Reply:
good q. I will investigate this and get back to you. might be a seperate error.

Reply:
this is expected behaviour. I updated the comments accordingly.